docs/Using-Fleet/REST-API.md

```diff
@@ -1014,6 +1022,8 @@ Modifies the Fleet's configuration with the supplied information.
 | api_token                         | string  | body  | _integrations.zendesk[] settings_. The Zendesk API token to use for this Zendesk integration. |
 | group_id                          | integer | body  | _integrations.zendesk[] settings_. The Zendesk group id to use for this integration. Zendesk tickets will be created in this group. |
 | apple_bm_default_team             | string  | body  | _mdm settings_. The default team to use with Apple Business Manager. |
+| minimum_version                   | string  | body  | _mdm.macos_updates settings_. Hosts that belong to no team and are enrolled into Fleet's MDM will be nudged until their macOS is at or above this version. | 
+| deadline                          | string  | body  | _mdm.macos_updates settings_. Hosts that belong to no team and are enrolled into Fleet's MDM won't be able to dismiss the Nudge window once this deadline is past. | 
```

Comment:
It should probably mention that this is fleet premium only? I see that it is missing for `apple_bm_default_team` above too. We did mention it for e.g. the transparency URL, so for consistency it should be there too I think.

Reply:
good catch! fixed in [ac8fbae](https://github.com/fleetdm/fleet/pull/9442/commits/ac8fbae03f93d5acceb27eb98324d1c3a68483b6)